platform.md

```diff
@@ -7,64 +7,114 @@ A platform orchestrates a lifecycle to make buildpack functionality available to
+
+The **build environment** refers to the containerized environment in which the lifecycle executes buildpacks.
+
+A **mixin** is a named set of additions to a stack that can be used to make additive changes to the contract. 
+
+An **app image** refers to an OCI image generated by the lifecycle by extending the **run image** with any or all of the following: **app layers**, **launch layers**, **launcher layers**, image configuration
 
-A **stack** is a contract defined by a base run OCI image and a base build OCI image that are only updated with security patches.
-Stack images can be modified with mixins in order to make additive changes to the contract.
+A **launch layer** refers to a layer in the app image created from a  `<layers>/<layer>` directory as specified in the [Buildpack Interface Specification](buildpack.md).
 
-A **mixin** is a named set of additions to a stack that avoid changing the behavior of buildpacks or apps that do not depend on the mixin.
+An **app layer** refers to a layer in the app image created from the `<app>` directory as specified in the [Buildpack Interface Specification](buildpack.md).
 
-A **launch layer** refers to a layer in the app OCI image created from a  `<layers>/<layer>` directory as specified in the [Buildpack Interface Specification](buildpack.md).
+A **run image layer** refers to a layer in the **app image** originating from the **run image**
 
-An **app layer** refers to a layer created from the `<app>` directory as specified in the [Buildpack Interface Specification](buildpack.md).
+A **launcher layer** refers to a layer in the app OCI image containing the launcher itself and/or launcher configuration
 
+The **launcher** refers to a lifecycle executable packaged in the app image for the purpose of executing processes at runtime
+
+#### Additional Terminology
+An **image reference** refers to either a **tag reference** or **digest reference**
+
+A **tag reference** refers to an identifier of form `<registry>/<repo>:<tag>` which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+
+A **digest reference**  refers to a [content addressable](https://en.wikipedia.org/wiki/Content-addressable_storage) identifier of form `<registry>/<repo>@<digest>` which locates an image manifest in an [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) compliant registry.
+
+The following is a non-exhaustive list of terms defined in the [OCI Image Format Specification](https://github.com/opencontainers/image-spec) used throughout this document:
+* **image config** https://github.com/opencontainers/image-spec/blob/master/config.md#oci-image-configuration
+* **imageID** - https://github.com/opencontainers/image-spec/blob/master/config.md#imageid
+* **DiffID** - https://github.com/opencontainers/image-spec/blob/master/config.md#layer-diffid
```

Comment:
```suggestion
* **diffID** - https://github.com/opencontainers/image-spec/blob/master/config.md#layer-diffid
```

Reply:
Do you want _both_ `ImageID` and `DiffID` capitalized or _neither_? The combination of this suggestion and https://github.com/buildpacks/spec/pull/87#discussion_r450564426 confuses me. I think the capitalization is a little awkward but fwiw they are both capitalized in the OCI spec.